src/EventListener/SlackNotifierEventListener.php

```diff
@@ -117,4 +125,19 @@ public function onCfpEndingSoon(CfpEndingSoonEvent $event): void
         $payload['attachments'][] = $event->buildAttachment();
         $this->slackNotifier->notify($payload);
     }
+
+    /**
+     * @param GenericEvent<EasyAdminEvents> $event
```

Comment:
Do this works? What does it mean?

Reply:
It works, it is just for phpstan. Since GenericEvent is an iterable, phpstan wants us to tell the iterable value type.
We could replace it with a phpstan-ignore-next-line but I think it is worse.

Reply:
Ok ok 👍🏼 (you don't iterate on it, but not pb :) )